db/schema.sql

```diff
@@ -0,0 +1,609 @@
+  (2, 12),
+  (2, 13),
+  (2, 14),
+  (2, 15),
+  (2, 19),
+  (2, 20),
+  (2, 21),
+  (2, 22),
+  (2, 23),
+  (2, 25),
+  (2, 26),
+  (2, 28),
+  (2, 29),
+
+  (3, 1),
+  (3, 4),
+  (3, 5),
+  (3, 6),
+  (3, 7),
+  (3, 9),
+  (3, 10),
+  (3, 16),
+  (3, 17),
+  (3, 18),
+  (3, 19),
+  (3, 20),
+  (3, 21),
+  (3, 22),
+  (3, 23),
+  (3, 25),
+  (3, 26),
+  (3, 29),
+  (3, 30),
+
+  (4, 1),
+  (4, 2),
+  (4, 7),
+  (4, 8),
+  (4, 9),
+  (4, 10),
+  (4, 11),
+  (4, 12),
+  (4, 13),
+  (4, 14),
+  (4, 15),
+  (4, 16),
+  (4, 17),
+  (4, 18),
+  (4, 19),
+  (4, 20),
+  (4, 21),
+  (4, 22),
+  (4, 26),
+  (4, 27),
+  (4, 28),
+  (4, 29),
+
+  (5, 4),
+  (5, 5),
+  (5, 6),
+  (5, 7),
+  (5, 8),
+  (5, 9),
+  (5, 10),
+  (5, 11),
+  (5, 12),
+  (5, 13),
+  (5, 14),
+  (5, 15),
+  (5, 16),
+  (5, 17),
+  (5, 18),
+  (5, 19),
+  (5, 20),
+  (5, 21),
+  (5, 22),
+  (5, 23),
+  (5, 24),
+  (5, 25),
+  (5, 26),
+  (5, 27),
+  (5, 29),
+  (5, 30),
+
+  (6, 1),
+  (6, 2),
+  (6, 3),
+  (6, 4),
+  (6, 5),
+  (6, 6),
+  (6, 7),
+  (6, 8),
+  (6, 9),
+  (6, 10),
+  (6, 11),
+  (6, 12),
+  (6, 16),
+  (6, 17),
+  (6, 18),
+  (6, 19),
+  (6, 20),
+  (6, 26),
+  (6, 27),
+  (6, 28),
+  (6, 29),
+  (6, 30),
+
+  (7, 1),
+  (7, 2),
+  (7, 3),
+  (7, 4),
+  (7, 5),
+  (7, 6),
+  (7, 7),
+  (7, 8),
+  (7, 9),
+  (7, 10),
+  (7, 11),
+  (7, 12),
+  (7, 13),
+  (7, 14),
+  (7, 15),
+  (7, 16),
+  (7, 23),
+  (7, 24),
+  (7, 25),
+  (7, 26),
+
+  (8, 1),
+  (8, 2),
+  (8, 3),
+  (8, 4),
+  (8, 5),
+  (8, 6),
+  (8, 13),
+  (8, 14),
+  (8, 15),
+  (8, 16),
+  (8, 17),
+  (8, 18),
+  (8, 19),
+  (8, 20),
+  (8, 21),
+  (8, 22),
+  (8, 23),
+  (8, 24),
+  (8, 25),
+  (8, 26),
+  (8, 27),
+  (8, 30),
+
+  (9, 1),
+  (9, 2),
+  (9, 3),
+  (9, 4),
+  (9, 5),
+  (9, 6),
+  (9, 7),
+  (9, 8),
+  (9, 9),
+  (9, 10),
+  (9, 11),
+  (9, 12),
+  (9, 13),
+  (9, 14),
+  (9, 15),
+  (9, 16),
+  (9, 17),
+  (9, 18),
+  (9, 19),
+  (9, 20),
+  (9, 21),
+  (9, 22),
+  (9, 23),
+  (9, 24),
+  (9, 25),
+  (9, 26),
+  (9, 27),
+  (9, 28),
+  (9, 29),
+  (9, 30),
+
+  (10, 1),
+  (10, 3),
+  (10, 4),
+  (10, 6),
+  (10, 7),
+  (10, 8),
+  (10, 11),
+  (10, 12),
+  (10, 13),
+  (10, 14),
+  (10, 15),
+  (10, 16),
+  (10, 18),
+  (10, 19),
+  (10, 20),
+  (10, 22),
+  (10, 23),
+  (10, 24),
+  (10, 26),
+  (10, 27),
+  (10, 28),
+  (10, 29),
+  (10, 30);
+
+# Tag
+
+INSERT INTO Tag (id, name) VALUES
+  (1, 'hot'),
+  (2, 'cold'),
+  (3, 'iced'),
+  (4, 'regular'),
+  (5, 'latte'),
+  (6, 'coffee'),
+  (7, 'tea'),
+  (8, 'classic'),
+  (9, 'Fizzio'),
+  (10, 'mixed'),
+  (11, 'frappuccino'),
+  (12, 'flakes'),
+  (13, 'best'),
+  (14, 'creamy'),
+  (15, 'delicious'),
+  (16, 'brewed'),
+  (17, 'blend'),
+  (18, 'energy'),
+  (19, 'espresso'),
+  (21, 'cappuccino'),
+  (22, 'lemon Ale'),
+  (23, 'fizzio'),
+  (24, 'real lemon'),
+  (25, 'juice'),
+  (26, 'ginger'),
+  (27, 'ginger Ale'),
+  (20, 'flowery');
+
+# OfferingTag
+
+INSERT INTO OfferingTag (offeringId, tagId) VALUES
+  (1, 1),
```

Comment:
same comment above on foreign key

Reply:
also makes it easier to read sql scripts

Reply:
Same as above. If the point was just to increase SQL readability, we shouldn't have bothered with normalization in the first place.